.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: required
 
+before_install:
+  - sudo apt-get update
```

Comment:
@petersowah Why did you make this change. As I said you shouldn't run `bundle update` like this.